querydsl-sql/src/test/java/com/querydsl/sql/types/TypeTest.java

```diff
@@ -45,7 +45,29 @@
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-        if (method.getName().startsWith("get")) {
+        if (method.getName().equals("wasNull")) {
```

Comment:
Yes, they have helper methods for a lot of things, it's knowing (and using) them that makes Guava so powerful.
However, the initial discoveries can be quite slow, since a lot of helper methods are very domain specific (`io` package is pretty common, but I don't use `net` for example)